src/framework/extensions/devtools/apidumpmodel.cpp

```diff
@@ -93,10 +93,11 @@ static QString moduleFromPrefix(const QString& prefix)
     return module;
 }
 
-bool ApiDumpModel::isAllowByType(const QString& module, ApiType type) const
+bool ApiDumpModel::isAllowedByType(const QString& module, ApiType type) const
 {
+#if 1
     return true;
```

Comment:
@igorkorsukov maybe this `return true` was not meant to be committed? Or maybe it's still work in progress?